<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>677</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Berlin Sans FB</family>
    <pointsize>49</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {	

	
	background-color: rgb(26, 65, 90);

}
/*

background-image: linear-gradient(15deg, #13547a 0%, #80d0c7 100%);
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #13547a, stop: .5 #80d0c7);
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0.4 #13547a, stop: 0.8 #80d0c7);
			

*/

/* background-color: rgb(26, 65, 90); */


</string>
  </property>
  <widget class="QLabel" name="Title_Screen">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee</family>
     <pointsize>32</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
boder:none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Instrução</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>69</y>
     <width>831</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* VERTIVAL SCROLL BAR*/
QScrollBar:vertical{
	boder: none;
	width: 14px;
	margin: 15px 0 15px 0;
	border-radius:10px;
	background-color: rgb(31, 125, 108);
}

/* HANDLE BAR VERTICAL */
QScrollBar::handle:vertical {
	border-radius:4px;
	min-height:15px;
	background-color: rgb(31, 79, 108);
}

/* BUTTON TOP - SCROLLBAR*/

QScrollBar::sub-line:vertical{
	border:none;	
	background-color: rgb(255, 255, 255);
	height: 15px;
	border-top-left-radius: 5px;
	border-top-right-radius:5px;
	subcontrol-position:top;
	subcontrol-origin: margin;
	
}

/* BUTTON BOTTOM - SCROLLBAR*/

QScrollBar::add-line:vertical{
	border:none;	
	background-color: rgb(255, 255, 255);
	height: 15px;
	border-bottom-left-radius: 5px;
	border-bottom-right-radius:5px;
	subcontrol-position:bottom;
	subcontrol-origin: margin;
}

/* RESET ARROW */

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {     
	 background: none;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: none;
}

QScrollBar::sub-page:vertical {
background: none;
}

QScrollBar::add-page:vertical {
background: none;
}

QWidget {

	background-color: transparent;

}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>511</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">/* VERTIVAL SCROLL BAR*/
QScrollBar:vertical{
	background-color: rgb(31, 125, 108);
}



</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-516</y>
       <width>817</width>
       <height>1553</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="slide1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>651</width>
          <height>501</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Images/Instrução_Slide_Menu.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="slide1_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>651</width>
          <height>501</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Images/Instrução_Slides_Edit_Screen01.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="slide1_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>651</width>
          <height>501</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Images/Instrução_Slides_Edit_Screen02.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="menu_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>610</y>
     <width>125</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background-color: white;
    border-style: outset;
    border-radius: 10px;    
    padding: 6px;
}
QPushButton:pressed {
    
	background-color: rgb(199, 199, 199);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>  Voltar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Images/left_arrow.png</normaloff>../Images/left_arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>Title_Screen</zorder>
  <zorder>menu_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
